packaging/hudi-spark-bundle/pom.xml

```diff
@@ -95,6 +95,12 @@
                   <include>org.antlr:stringtemplate</include>
                   <include>org.apache.parquet:parquet-avro</include>
 
+                  <include>com.fasterxml.jackson.core:jackson-annotations</include>
+                  <include>com.fasterxml.jackson.core:jackson-core</include>
+                  <include>com.fasterxml.jackson.core:jackson-databind</include>
+                  <include>com.fasterxml.jackson.dataformat:jackson-dataformat-yaml</include>
+                  <include>com.fasterxml.jackson.module:jackson-module-scala_${scala.binary.version}</include>
```

Comment:
wondering why we add this?

Reply:
Is this for fixing testing only?  We should avoid introducing new changes for production code and bundling.  If really necessary, could you add these to test scope only or integ-test-bundle?